woocommerce/woocommerce-gutenberg-products-block
# Use real previews for the Cart and Checkout blocks

Part of #2981 

This PR uses real previews for the cart and checkout, something we can use because:
1- We have container queries so our blocks are responsive regardless of the screen size.
2- We don't have a select product step for those blocks, what you see is what you get.

### Screenshots
Before:
![image](https://user-images.githubusercontent.com/6165348/89906843-cc4ed900-dbe3-11ea-9e99-6432a8649eb0.png)
![image](https://user-images.githubusercontent.com/6165348/89906885-d7a20480-dbe3-11ea-811c-23b5b34614ed.png)

<img width="280" alt="Screen Shot 2020-08-11 at 1 47 46 PM" src="https://user-images.githubusercontent.com/6165348/89906488-5ea2ad00-dbe3-11ea-838f-ec7d5ac2bc83.png">
<img width="283" alt="Screen Shot 2020-08-11 at 1 48 43 PM" src="https://user-images.githubusercontent.com/6165348/89906495-606c7080-dbe3-11ea-99ab-35aa4b3ec896.png">

After:
![image](https://user-images.githubusercontent.com/6165348/89906720-a6293900-dbe3-11ea-959f-4cb5f3053c12.png)
![image](https://user-images.githubusercontent.com/6165348/89906754-ade8dd80-dbe3-11ea-9884-6fe246bc1527.png)

<img width="280" alt="Screen Shot 2020-08-11 at 2 40 32 PM" src="https://user-images.githubusercontent.com/6165348/89906523-69f5d880-dbe3-11ea-8f9c-d8d88126a748.png">
<img width="283" alt="Screen Shot 2020-08-11 at 2 40 38 PM" src="https://user-images.githubusercontent.com/6165348/89906537-6d895f80-dbe3-11ea-856e-aac149fbbc06.png">




### How to test the changes in this Pull Request:

1. Preview the cart and checkout inside the block inserter.
2. Make sure the block is not very long and overflowing.
3. Make sure the changes didn't leak to the actual block or the editor block.



### Changelog

> Cart and Checkout will show a live preview inside the block inserter and style selector.
